src/lib.rs

```diff
@@ -631,12 +680,48 @@ fn create_task_worker(
     })
 }
 
+/// Spawn a tokio task to perform the burn_in_test generation work.
+///
+/// # Arguments
+///
+/// * `deps` - Service dependencies.
+/// * `build_variant` - Build variant to query timing information from.
+/// * `generated_tasks` - Map to stored generated to in.
```

Comment:
[nit] Not getting this part `generated to in`

Reply:
fixed.